machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -418,4 +422,74 @@ func main() {
 			log.ErrorExit(err)
 		}
 	}
+
+	if *flgBMCLogCollectorConfigMap {
+		// create bmc-log-collector configmap of all servers that include boot servers
+		addersses, err := updateServerAddressList(machines)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+		err = client.writeConfigMap(ctx, addersses)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+	}
+}
+
+func updateServerAddressList(machines []Machine) (map[string]string, error) {
+	type MachineConfigMap struct {
```

Comment:
Is this struct name `MachineConfigMap` appropriate? Is this a ConfigMap?

Reply:
Changed MachinesList